### PR TITLE
Colw/mobile signin

### LINKED ISSUE #2885: Reduce mobile account options
**Is your feature request related to a problem? Please describe.**

For now we want to reduce the options on mobile to explore, local sign in, account creation and account import. Extension and Ledger are not available on mobile.

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by colw**
Closes #2885 

**Description:**

When in mobile app mode we set insecure mode to true, and hide ledger, extension options.

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by faboweb** [comment]
I think we still need to sign in, into a created address.

File: src/components/common/TmSessionExisting.vue [hunk 1]
```diff
@@ -13,13 +13,14 @@
...
         <LiSession
+          v-if="!isMobile"
           id="use-ledger-nano"
           icon="vpn_key"
           title="Use Ledger Nano"
           route="ledger"
         />
         <LiSession
-          v-if="session.experimentalMode || extension.enabled"
+          v-if="session.experimentalMode || extension.enabled || !isMobile"
```
Comment (faboweb) [inline src/components/common/TmSessionExisting.vue]:
```suggestion
          v-if="(session.experimentalMode || extension.enabled) && !isMobile"
```

Comment (colw) [reply src/components/common/TmSessionExisting.vue]: Cheers, yep got this 👍

**Comment by jbibla** [comment]
> insecure mode to true

let's change the name of this mode because this is bad vibes 🙊

**Comment by faboweb** [comment]
> > insecure mode to true
> 
> let's change the name of this mode because this is bad vibes 🙊

can we do it in a different pr?

**Pull request merged**